dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryAPIImpl.java

```diff
@@ -125,7 +124,10 @@ public Category find(String id, User user, boolean respectFrontendRoles) throws
 
 	public void save(Category parent, Category object, User user, boolean respectFrontendRoles) throws DotDataException, DotSecurityException {
 
-		boolean isANewCategory = false;
+        // Checking that we have a unique key.
+	    object = checkUniqueKey(object, user);

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "object". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1226)